spec/services/catalog/update_order_item_spec.rb

```diff
@@ -1,7 +1,7 @@
 describe Catalog::UpdateOrderItem do
   describe "#process" do
     let(:client) { double(:client) }
-    let(:topic) { ManageIQ::Messaging::ReceivedMessage.new(nil, nil, payload, nil, client) }
+    let(:topic) { ManageIQ::Messaging::ReceivedMessage.new(nil, nil, payload, nil, client, nil) }
```

Comment:
@syncrou Is this related to this PR or a fix for some other PR. Could it be this one.
https://github.com/ManageIQ/manageiq-messaging/pull/46
If its un-related can we put it in a different PR